docs/dev/core/internal/debug-pings.md

```diff
@@ -0,0 +1,15 @@
+# Debug Pings
+
+For debugging and testing purposes Glean allows to tag pings, which are then available in the [Debug Ping Viewer][debug-ping-viewer].
+
+Pings are sent to the same endpoint as all pings, with the addition of one HTTP header:
+
+```
+X-Debug-ID: <tag>
+```
+
+`<tag>` is a alphanumeric string with a maximum length of 20 characters, used to identify pings in the Debug Ping Viewer.
+
+See [Debugging products using the Glean SDK](/user/debugging/index.md) for detailed information how to use this mechanism in applications.
+
+[debug-ping-viewer]: https://glean-debug-view-dev-237806.firebaseapp.com/
```

Comment:
Should we mention this is currently only available to those with a Mozilla login?